### ISSUE TITLE
Не очень короткое введение

### ISSUE THREAD

**Issue by kia999** [odomanov/biblatex-gost#32]
Здравствуйте, Олег! 
Признаю: Biblatex -- очень мощная и гибкая система, гораздо более гибкая, чем bibtex. Yо документация не рассчитана на начинающего пользователя. Читать 32 страницы найдётся немного желающих. Это в равной степени относится как к пакету biblatex-gost, так и biblatex. Нельзя ли сделать вводный раздел с парой темплэйстов? Что людям нужно от biblatex-gost? -- оформить диссертацию по ГОСТу или книгу.

**Comment by odomanov** [comment]
Да, я согласен, хорошо бы это написать, но руки не доходят. Я, кстати, и на английском не видел хорошего введения. Есть хорошая французская книга, я даже думал её перевести, но опять же -- время нужно.
Можно попробовать FAQ на сайте сделать, это, наверное, проще.
